### DISCUSSION TITLE
Collapse Transition, Duration, and Timing

### DISCUSSION THREAD

**Discussion by yoelpc4** [themesberg/flowbite#149] [Q&A]
I read you write on docs about customizing of collapse transition, duration, and timing [here](https://flowbite.com/docs/components/navbar/)

But after digging deeper into the class there's no way to customize it, is there any way to set animation with a duration of maybe 500ms slide left and right?

**Comment by mbhnyc** [comment]
yeah looking at the source code, only the callbacks appear as options, @yoelpc4 did you ever figure out a way to animate this?
